mkdocs.yml

```diff
@@ -0,0 +1,69 @@
+site_name: Flux
+# site_description:
+site_author: Flux Project
```

Comment:
Missed this last night, I changed this to  `The <project name> contributors` for the Helm operator as this seems to be inline with the ecosystem and CNCF projects.